client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/index.tsx

```diff
@@ -63,16 +63,16 @@ const VerticalSelect: React.FunctionComponent = () => {
 
 	const animatedPlaceholder = useTyper(
 		[
-			NO__( 'photography' ),
-			NO__( 'blogging' ),
-			NO__( 'travel' ),
-			NO__( 'marketing' ),
-			NO__( 'fashion' ),
-			NO__( 'shopping' ),
-			NO__( 'design' ),
-			NO__( 'real estate' ),
-			NO__( 'food' ),
-			NO__( 'sports' ),
+			__( 'photography' ),
+			__( 'blogging' ),
+			__( 'travel' ),
+			__( 'marketing' ),
+			__( 'fashion' ),
+			__( 'shopping' ),
+			__( 'design' ),
+			__( 'real estate' ),
+			__( 'food' ),
+			__( 'sports' ),
```

Comment:
These would need explaining that they'll be in sentence like `Itʼs called <Input />`

Reply:
776ff14fcabe34655996988fb6382c823e8a9b38